### ISSUE TITLE
Cannot source generate CRD models because of error `Could not load file or assembly YamlDotNet`

### ISSUE THREAD

**Issue by ddadaal** [IvanJosipovic/KubernetesCRDModelGen#273]
Hello, I am trying to use the source generator in my project. I added the following lines in my csproj file

```xml
    <PackageReference Include="KubernetesCRDModelGen.SourceGenerator" Version="1.0.0-alpha.168" />
    <AdditionalFiles Include="<the path to my crd yaml" />
```

and run `dotnet build`, and then an error occurred 

```
CSC : warning CS8785: Generator 'Generator' failed to generate source. It will not contribute to the output and compilation errors may occur as a result. Exception was of type 'FileNotFoundExcep 
tion' with message 'Could not load file or assembly 'YamlDotNet, Version=13.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e'. The system cannot find the file specified.'
```

I tried to manually add `YamlDotNet` to my csproj but it didn't help. From the [nuget site](https://www.nuget.org/packages/KubernetesCRDModelGen.SourceGenerator/1.0.0-alpha.168#dependencies-body-tab), the source generator doesn't have the dependency, but the dependency does show up in the csproj file of the source generate project. Is there any problem here?

Thanks!

**Comment by IvanJosipovic** [comment]
Hey,

Thanks for the report, this should be fixed in [1.0.0-alpha.169](https://www.nuget.org/packages/KubernetesCRDModelGen.SourceGenerator/1.0.0-alpha.169)

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
    <LangVersion>latest</LangVersion>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="KubernetesCRDModelGen.SourceGenerator" Version="1.0.0-alpha.169" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
    <AdditionalFiles Include="*.yaml" />
  </ItemGroup>

</Project>

```

Let me know how it goes!

**Comment by ddadaal** [comment]
Thanks! It works now.
